src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -116,6 +116,16 @@ public function prepend(ContainerBuilder $container)
 
         $container->prependExtensionConfig('doctrine', $doctrineGedmoEntityConfig);
 
+        $frameworkExtensionConfig = [
+            'assets' => [
+                'packages' => [
+                    'asset' => null,
```

Comment:
I'm thinking, do we need the asset config? Can we use the asset function without the package parameter for all calls in the code, except for the media assets. Which can use the "media" package. Is that something that could work?

Reply:
@acrobat That works too. I have updated the PR.